### ISSUE TITLE
Events

### ISSUE THREAD

**Issue by Lee182** [akxcv/vuera#54]
Vue in React, how would 1 handle events from a vue component `vm.$emit('eventnnaeme', stuff)`

**Comment by akxcv** [comment]
Hi, what do you mean, exactly? Do you want to send events from Vue to React, or from Vue to [Vue in React]?

**Comment by Lee182** [comment]
Something like this. [Vue in React] is exactly what i mean.
```js
// React file
class Hello extends React.Component {
  constructor(super) {
    super(props)
  }
  doSomthingReact() {
  }
  render() {
    <MyVueComponent onChange={this.doSomethingReact.bind(this)} />
  }
}
```

```vue
<template>
<button @click='onClick'>This is a complex button</button>
</template>
<script>
// my vue component
export default {
  methods: {
    onClick() {
      vm = this
      vm.$emit('change', {someComplexBusinessLogicResult: '...'})
    }
  }
}
</script>
```

**Comment by akxcv** [comment]
Well, React doesn't have Vue-like event emitting out of the box. In your case, it's best to use a custom event emitter. This is not a responsibility of vuera.

**Comment by Lee182** [comment]
What i'm on about is handling vue component events in react. The react way of doing things is to pass a callback down via props. I'm unsure how else would you catch and handle these events, especially if using external modules where you can't change the code to use a custom event emitter.

**Comment by akxcv** [comment]
Can't you just call the `onChange` prop from the Vue component?

**Comment by Lee182** [comment]
I guess you would have to write a separate Vue component as a wrapper which takes the `onChange` prop and any other props from react and then handle the event to bind to the callback on the vue end. be it `v-on:change @change` template syntax or using something like vue-jsx babel plugin simply `onChange`, i was just wondering if there was a way to do this in vuera parent-child communication.

**Comment by akxcv** [comment]
Parent-child communication isn't something vuera currently concerns itself with. Perhaps it should, but we'll have to think about the API for that. If you could provide an example of how you see it working, that would be amazing.

**Comment by Lee182** [comment]
Yeah sure, I'm glad you might implement this
```js 
// My React render method
return <MyVueComponent
someProp={this.state.sompProp}
someOtherProp={this.state.someOtherProp}
on={{
  change: this.handleChange.bind(this),
  loading: this.handleLoading.bind(this)
}} />
```
```js
// some where in the vue component
vm.$emit('change', {})
vm.$emit('loading', true)
```

**Comment by akxcv** [comment]
Closed by #62
